client/miner.c

```diff
@@ -412,54 +414,15 @@ static void *mining_thread(void *arg)
 	return 0;
 }
 
-static void *general_mining_thread(void *arg)
-{
-	while(!g_xdag_sync_on && !g_stop_general_mining) {
-		sleep(1);
-	}
-
-	while(!g_stop_general_mining) {
-		xdag_create_block(0, 0, 0, 0, xdag_main_time() << 16 | 0xffff, NULL);
-	}
-
-	xdag_mess("Stopping general mining thread...");
-
-	return 0;
-}
-
 /* changes the number of mining threads */
 int xdag_mining_start(int n_mining_threads)
 {
 	pthread_t th;
 
```

Comment:
Why need move general_mining_thread to pool.c?

Reply:
This thread generates new blocks. So it is related to pool.